tensorflow/python/ops/array_ops.py

```diff
@@ -3124,6 +3124,10 @@ def squeeze_v2(input, axis=None, name=None):
 
 
 @tf_export("where")
```

Comment:
I'm surprised this works at all. This should be v1=["where"] (since there's another export for v2)

@annarev shouldn't this export conflict with the one below and throw an error in the api generation?